src/App.jsx

```diff
@@ -14,57 +16,57 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood === null ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood} is selected
+          <button data-cy="ClearButton" type="button" className="delete ml-3" />
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr data-cy="Good" key={good}>
```

Comment:
add class `has-background-success-light` to tr for selected good.
it will help to pass test.
![Screenshot 2024-04-06 at 20 14 47](https://github.com/mate-academy/react_goods-selector-js/assets/33029864/c2a484fa-525f-412d-aee2-83d13ca94874)

working example:
![Screenshot 2024-04-06 at 20 15 27](https://github.com/mate-academy/react_goods-selector-js/assets/33029864/458b3a9e-c9ea-4b47-88b3-0f5e372b20ca)

actual:
![Screenshot 2024-04-06 at 20 16 11](https://github.com/mate-academy/react_goods-selector-js/assets/33029864/9b8b8cc6-a4ee-4de8-a28e-43b8763e9ff2)